Error: Cannot find module 'nodejs-dashboard'

mac OS 10.13.3
node v 8.9.1
Tried on Hyper and iTerm
```
npm install -g nodejs-dashboard
nodejs-dashboard -- node -r nodejs-dashboard test.js
```

This starts the dashboard but gives me an error accessing the module...

<img width="571" alt="screen shot 2018-03-29 at 14 06 53" src="https://user-images.githubusercontent.com/1792811/38090443-7f281ecc-335a-11e8-9fd5-4c042cb42b92.png">

The `-r nodejs-dashboard` part **is** required and is the only require that we need for the dashboard. The problem is that sometime after like node8 or what not globally installed things via at least NVM which I use aren't findable.

Here's some info:

```sh
$ npm install -g nodejs-dashboard
$ node
> const dash = require("nodejs-dashboard");
Thrown:
{ Error: Cannot find module 'nodejs-dashboard'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18) code: 'MODULE_NOT_FOUND' }
```

When I dig and prefix the path to the actual nodejs-dashboard module, it works:

```sh
$ nodejs-dashboard -- nodemon -r "/Users/rye/.nvm/versions/node/v10.16.3/lib/node_modules/nodejs-dashboard" src/index.js 
```

I will try and see if I can get a more general solution...

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,10 @@
 # Change log
 
+## UNRELEASED
+
+- **Fixed**: Support global installs of `nodejs-dashboard`. Add the `node_modules` directory from wherever `nodejs-dashboard` is installed to `NODE_PATH` to support `node -r nodejs-dashboard` required for full usage. [\#90]
+- *Internal*: Use SIGINT for Ctrl-c. [\#93]
+
 ## [v0.5.0] - 2019-11-21
 
 - **Breaking**: Update node engines to `8`+
@@ -57,6 +62,7 @@
 - *Internal*: Remove dependency on root-require, update repo url in package.json [\#2]
 - *Internal*: Test scaffolding and basic reporter integration test [\#3]
 
+[v0.5.0]: https://github.com/FormidableLabs/nodejs-dashboard/compare/v0.4.3...v0.5.0
 [v0.4.3]: https://github.com/FormidableLabs/nodejs-dashboard/compare/v0.4.1...v0.4.3
 [v0.4.1]: https://github.com/FormidableLabs/nodejs-dashboard/compare/v0.4.0...v0.4.1
 [v0.4.0]: https://github.com/FormidableLabs/nodejs-dashboard/compare/v0.3.0...v0.4.0
@@ -66,6 +72,8 @@
 [v0.1.2]: https://github.com/FormidableLabs/nodejs-dashboard/compare/v0.1.1...v0.1.2
 [v0.1.1]: https://github.com/FormidableLabs/nodejs-dashboard/compare/v0.1.0...v0.1.1
 
+[\#93]: https://github.com/FormidableLabs/nodejs-dashboard/pull/93
+[\#90]: https://github.com/FormidableLabs/nodejs-dashboard/issues/90
 [\#76]: https://github.com/FormidableLabs/nodejs-dashboard/pull/76
 [\#68]: https://github.com/FormidableLabs/nodejs-dashboard/pull/72
 [\#67]: https://github.com/FormidableLabs/nodejs-dashboard/pull/70

--- a/bin/nodejs-dashboard.js
+++ b/bin/nodejs-dashboard.js
@@ -75,9 +75,23 @@ process.env[config.PORT_KEY] = port;
 process.env[config.REFRESH_INTERVAL_KEY] = program.refreshinterval;
 process.env[config.BLOCKED_THRESHOLD_KEY] = program.eventdelay;
 
+// Enhance `NODE_PATH` to include the dashboard such that `require("nodejs-dashboard")`
+// works even if globally installed.
+// See: https://github.com/FormidableLabs/nodejs-dashboard/issues/90
+const IS_WIN = process.platform.indexOf("win") === 0;
+const DELIM = IS_WIN ? ";" : ":";
+const DASHBOARD_PATH = path.resolve(__dirname, "../..");
+const NODE_PATH = (process.env.NODE_PATH || "")
+  .split(DELIM)
+  .filter(Boolean)
+  .concat(DASHBOARD_PATH)
+  .join(DELIM);
 
 const child = spawn(command, args, {
-  env: process.env,
+  env: {
+    ...process.env,
+    NODE_PATH
+  },
   stdio: [null, null, null, null],
   detached: true
 });

--- a/lib/dashboard.js
+++ b/lib/dashboard.js
@@ -52,7 +52,7 @@ Dashboard.prototype._configureKeys = function () {
   // this key will be watched on the global screen
   this.screen.ignoreLocked = ["C-c"];
   this.screen.key("C-c", () => {
-    process.exit(0); // eslint-disable-line no-process-exit
+    process.kill(process.pid, "SIGINT");
   });
 
   // watch for key events on the main container; not the screen

